arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTraceOptionsViewController.swift

```diff
@@ -256,3 +256,24 @@ class ConfigureSubnetworkTraceOptionsViewController: UITableViewController {
         }
     }
 }
+
+/// An extension of `AGSUtilityAttributeComparisonOperator` that returns a human readable description.
+private extension AGSUtilityAttributeComparisonOperator {
+    static let allCases = (Self.equal.rawValue...Self.doesNotIncludeAny.rawValue).map { AGSUtilityAttributeComparisonOperator(rawValue: $0)! }
```

Comment:
While this saves some typing, I'm wondering if it would be more straightforward to just use an array literal with each of the cases listed. I think that's what we've done elsewhere. An approach like this would be more useful if it was future-proof (ie. automatically included any new cases added in subsequent release), but it isn't.